domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/Claim.java

```diff
@@ -196,6 +200,10 @@ public LocalDate getServiceDate() {
         return TotalAmountCalculator.calculateInterestForClaim(this);
     }
 
+    public List<String> getFeatures() {
```

Comment:
Good we do not have to check equals and hashCode anymore :) Thanks to @sabahirfan